src/app/store/store.module.ts.template

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2018 University of Stuttgart.
```

Comment:
I guess we should update this to be 2018-2021.

Reply:
Done.